docs/processes-v2/migration.md

```diff
@@ -234,6 +234,58 @@ flows:
         method: "GET" # error: 'url' is required
 ```
 
+## Task Parameters
+
+#### Removed Input Parameters
+
+- `out`: no longer required to specify an out-variable name
+
+#### Removed Output Values
+
+- `success`: Use standardized ok value
+- `errorString`: Use standardized error value
```

Comment:
These parameter changes are specific to the `http` task.

Reply:
oops